xarray/core/variable.py

```diff
@@ -542,6 +547,8 @@ def _dask_finalize(self, results, array_func, *args, **kwargs):
     @property
     def values(self):
         """The variable's data as a numpy.ndarray"""
+        if isinstance(self._data, PandasExtensionArray):
+            return self._data.array
         return _as_array_or_item(self._data)
```

Comment:
I think we want to preserve the `np.ndarray` return type here.
```suggestion
    def values(self) -> np.ndarray:
        """The variable's data as a numpy.ndarray"""
        return _as_array_or_item(self._data)
```

Reply:
Sorry, didn't read the comment!